otterdog/eclipse-keypop.jsonnet

```diff
@@ -42,6 +44,8 @@ orgs.newOrg('eclipse-keypop') {
       allow_update_branch: false,
       web_commit_signoff_required: false,
     },
+    orgs.newRepo('keypop-calypso-crypto-symmetric-cpp-api') {
+    },    
```

Comment:
here is trailing white space that could be removed to make it equal to the canonical version.